nva-datamodel-java/src/main/java/no/unit/nva/model/PublicationStatus.java

```diff
@@ -11,6 +11,7 @@ public enum PublicationStatus {
     DRAFT("DRAFT"),
     PUBLISHED_METADATA("PUBLISHED_METADATA"),
     PUBLISHED("PUBLISHED"),
+    DELETED("DELETED"),
```

Comment:
So it's clear, "Deleted" implies that 
- there was a Published resource that is now marked as deleted and should not be understood as published
- it does not appear in any search index
- its landing page is a "tombstone", stating that it has ceased to be

Reply:
Also see thread in https://sikt-no.slack.com/archives/CN005AQCE/p1673430958877679.